timesync/config/config.go

```diff
@@ -0,0 +1,51 @@
+package config
+
+import (
+	"time"
+
+	"github.com/spacemeshos/go-spacemesh/log"
+)
+
+// ConfigValues specifies  default values for node config params.
+var (
+	ConfigValues     = DefaultConfig()
+	TimeConfigValues = ConfigValues.TimeConfig
+)
+
+// TimeConfig specifies the timesync params for ntp.
+type TimeConfig struct {
+	MaxAllowedDrift       time.Duration `mapstructure:"max-allowed-time-drift"`
+	NtpQueries            int           `mapstructure:"ntp-queries"`
+	DefaultTimeoutLatency time.Duration `mapstructure:"default-timeout-latency"`
+	RefreshNtpInterval    time.Duration `mapstructure:"refresh-ntp-interval"`
+}
+
+// Config defines the configuration options for the Spacemesh peer-to-peer networking layer
+type Config struct {
```

Comment:
why put that under another struct ? just use `TimeConfig`

Reply:
I did it following the same convention as on api and p2p packages. It's true that is not necessary, so I  removed it.